CHANGELOG.md

```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
- 
+
+## [1.12.2] - 2021-04-01 
+### Fixed
+- Wildcard configuration issues
```

Comment:
What are these issues ?

Reply:
![image](https://user-images.githubusercontent.com/9078616/113300824-a1460080-92fe-11eb-9a51-1ca4e4fa1d3b.png)
crash when "*" is seen in traded pairs config